todo/urls.py

```diff
@@ -0,0 +1,31 @@
+from django.urls import path
+
+from .views import (
```

Comment:
Why can't we create two separate files called _task_views.py_ and _tag_views.py_, each containing the relevant views for tasks and tags respectively? Then, in your urls.py file, you could import these views and use them in your URL patterns, for example:
`from . import task_views, tag_views

urlpatterns = [
    path("task/<int:pk>/update", task_views.TaskUpdateView.as_view(), name="task-update"),
    path("tags/<int:pk>/delete", tag_views.TagDeleteView.as_view(), name="tag-delete"),
    path('chng_st/<int:pk>', task_views.TaskChangeStatusView.as_view(), name='task_change_status'),
]`

This approach allows you to separate concerns and keep related code together in separate modules, which can make it easier to maintain and modify the code in the future.